environment.yml

```diff
@@ -0,0 +1,34 @@
+name: geopandas-dev
+channels:
+    - conda-forge
+dependencies:
+    # required
+    - Cython>=0.16
```

Comment:
You can remove this line, we actually don't need cython

Reply:
Cool, I couldn't see why that was actually required, but also guessed it wouldn't have been removed as a dependency where it had been before.